app/assets/stylesheets/ror-widget.css

```diff
@@ -0,0 +1,31 @@
+.institution-input-disable {
+    background-color: #EEEEEE !important;
+    cursor: not-allowed;
+}
+
+.tt-menu {
+    width: 422px;
+    margin: 12px 0;
+    padding: 8px 0;
+    background-color: #fff;
+    border: 1px solid #ccc;
+    max-height: 400px;
+    overflow-y: auto;
+    border: 1px solid rgba(0, 0, 0, 0.11);
+    -webkit-border-radius: 8px;
+    -moz-border-radius: 8px;
+    border-radius: 8px;
+    -webkit-box-shadow: 0 5px 10px rgba(0,0,0,.2);
+    -moz-box-shadow: 0 5px 10px rgba(0,0,0,.2);
+    box-shadow: 0 5px 10px rgba(0,0,0,.2);
+
+
+}
+
+.league-name {
+    margin: 0 20px 5px 0px;
+    padding: 3px 6px;
+    background-color: rgba(177, 183, 187, 0.47);
+    transition: background-color 0.5s ease-in-out;
```

Comment:
Try and use bootstrap sass variables for the colours, otherwise it will look weird on instances that are using a different theme, like we do here: 
https://github.com/seek4science/seek/blob/main/app/assets/stylesheets/styles.scss#L440-L443
e.g. you could use `$input-bg` for the `background-color`
https://github.com/twbs/bootstrap-sass/blob/v3.4.3/assets/stylesheets/bootstrap/_variables.scss#L189-L192